/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-desktop-icon.1 xdg-utils-1.0.2/scripts/man/xdg-desktop-icon.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-desktop-icon.1	2009-02-19 17:44:33.548463000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-desktop-icon.1	2009-02-19 17:49:58.314003000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -115,6 +116,21 @@ The action failed.
 .TP
 \fB5\fR
 No permission to read one of the files passed on the command line.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 \fBxdg\-icon\-resource\fR(1)
@@ -156,3 +172,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-desktop-menu.1 xdg-utils-1.0.2/scripts/man/xdg-desktop-menu.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-desktop-menu.1	2009-02-19 17:44:33.548282000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-desktop-menu.1	2009-02-19 17:50:04.880781000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -199,6 +200,21 @@ The action failed.
 .TP
 \fB5\fR
 No permission to read one of the files passed on the command line.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 \fBxdg\-desktop\-icon\fR(1),
@@ -321,3 +337,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-email.1 xdg-utils-1.0.2/scripts/man/xdg-email.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-email.1	2009-02-19 17:44:33.548963000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-email.1	2009-02-19 17:50:51.750103000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -90,6 +91,21 @@ No permission to read one of the files p
 .SH "CONFIGURATION"
 .PP
 Visit http://portland.freedesktop.org/EmailConfig for information how to configure xdg\-email to use the email client of your choice.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "EXAMPLES"
 .PP
 .nf
@@ -116,3 +132,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-icon-resource.1 xdg-utils-1.0.2/scripts/man/xdg-icon-resource.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-icon-resource.1	2009-02-19 17:44:33.548638000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-icon-resource.1	2009-02-19 17:50:59.273929000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -136,6 +137,21 @@ The action failed.
 .TP
 \fB5\fR
 No permission to read one of the files passed on the command line.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 \fBxdg\-desktop\-icon\fR(1),
@@ -164,3 +180,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-mime.1 xdg-utils-1.0.2/scripts/man/xdg-mime.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-mime.1	2009-02-19 17:44:33.548059000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-mime.1	2009-02-19 17:51:02.158958000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -137,6 +138,21 @@ The action failed.
 .TP
 \fB5\fR
 No permission to read one of the files passed on the command line.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "SEE ALSO"
 .PP
 \fBxdg\-icon\-resource\fR(1),
@@ -186,3 +202,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-open.1 xdg-utils-1.0.2/scripts/man/xdg-open.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-open.1	2009-02-19 17:44:33.548791000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-open.1	2009-02-19 17:51:08.820623000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -44,6 +45,21 @@ A required tool could not be found.
 .TP
 \fB4\fR
 The action failed.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "EXAMPLES"
 .PP
 .nf
@@ -63,3 +79,4 @@ Kevin Krammer, Jeremy White. 
 <kevin.krammer@gmx.at>
 .br
 <jwhite@codeweavers.com>
+...\" LSARC 2008/104 XDG Utils
/jds/bin/diff -uprN xdg-utils-1.0.2.old/scripts/man/xdg-screensaver.1 xdg-utils-1.0.2/scripts/man/xdg-screensaver.1
--- xdg-utils-1.0.2.old/scripts/man/xdg-screensaver.1	2009-02-19 17:44:33.549120000 +0000
+++ xdg-utils-1.0.2/scripts/man/xdg-screensaver.1	2009-02-19 17:51:18.492671000 +0000
@@ -1,3 +1,4 @@
+'\" te
 .\" ** You probably do not want to edit this file directly **
 .\" It was generated using the DocBook XSL Stylesheets (version 1.69.1).
 .\" Instead of manually editing it, you probably should edit the DocBook XML
@@ -79,6 +80,21 @@ A required tool could not be found.
 .TP
 \fB4\fR
 The action failed.
+
+.SH "ATTRIBUTES"
+.PP
+See \fBattributes\fR(5)
+for descriptions of the following attributes:
+.sp
+.TS
+tab(^G) allbox;
+cw(2.750000i)| cw(2.750000i)
+lw(2.750000i)| lw(2.750000i).
+ATTRIBUTE TYPE^ATTRIBUTE VALUE
+Availability^SUNWxdg-utils
+Interface stability^Uncommitted
+.TE
+.sp
 .SH "EXAMPLES"
 .PP
 .nf
@@ -94,3 +110,4 @@ must be the X Window ID of an existing w
 Bryce Harrington. 
 .br
 <bryce@osdl.org>
+...\" LSARC 2008/104 XDG Utils
